Makefile

```diff
@@ -1,2 +0,0 @@
-serve:
```

Comment:
It would be nice to keep the Makefile for middleman server and build

Reply:
Good point, feel free to add a Makefile back in. Also we need to discuss the best way to automate the build/publishing.

Reply:
Added Makefile back in with `server` and `build` commands